hooks/tk-multi-loader2/basic/scene_actions.py

```diff
@@ -13,12 +13,18 @@
 """
 import os
 
+try:
+    import builtins
+except ImportError:
+    import __builtins__ as builtins
+
 import sgtk
 
 from vrKernelServices import vrSceneplateTypes
 from vrKernelServices import vrdSceneplateNode
 import vrFileIO
-import vrScenegraph
+
+builtins.vrReferenceService = vrReferenceService
```

Comment:
These two hound errors should be ignored because of this line:

https://github.com/shotgunsoftware/tk-vred/blob/6564cfd716b5490145fcc81305ea509e8ac03982/.flake8#L52

Weird!